### DISCUSSION TITLE
Missing cover arts :(

### DISCUSSION THREAD

**Discussion by gerroon** [ampache/ampache#3667] [Q&A]
Hi guys

I do not understand why Ampache fails to find cover arts. Here is an example of what Ampache has and what the folder has. I have coverart in both places.

![image](https://github.com/ampache/ampache/assets/8519469/2cb173b2-f3ed-4df0-8e55-2bd6883c42b1)
In case it is finding the album here since we do not have file paths in Amapche I do noy know what comes from where. These are the two references.
![image](https://github.com/ampache/ampache/assets/8519469/7263cbaa-434a-4818-95b1-9261aac6362e)

![image](https://github.com/ampache/ampache/assets/8519469/b1750fe9-c760-4921-9dbd-e32de0763412)

**Comment by kuzi-moto** [comment]
Have you already tried running a manual “gather art” command? https://github.com/ampache/ampache/wiki/Catalog

**Reply by gerroon** [reply]
Yeah I tried it already. It really gathered art, but not for all. I am still tryin got figure it out,. Has anyone had similar issues with it?

**Reply by mitchray** [reply]
What is your setting for ```album_art_store_disk``` in the config file (double check if its commented out)

**Comment by rhauff1** [comment]
Yes, mine stopped gathering cover art somewhere in version 5.  When I updated to version 6 it gathered all art correctly that had been missing for 6 months or so.  However, when I add music to version  6.0.3 now via Catalog>Add, it is again not loading the artwork.  I have my config set to load from metadata first and all my music had images embedded.  I can use the individual album Cover Art Edit, and the first image appears to be the metadata image so I select that and save.  Don't understand why I have to manually pick it.  I have not tried the Catalog "Gather 
Art" yet on V6.
![CoverArt_edit](https://github.com/ampache/ampache/assets/75855702/5998f8a1-b546-42f3-baa9-439b13497968)
![CoverArt_Select](https://github.com/ampache/ampache/assets/75855702/a0688a87-9aff-4ac2-9cd8-1f3434a54f77)

**Reply by mitchray** [reply]
What is your setting for ```album_art_store_disk``` in the config file (double check if its commented out)

**Reply by rhauff1** [reply]
Well, it is commented out by default & that's how mine is.
; Album Art Store on Disk
; This defines if arts should be stored on disk instead of database.
; You must also set a local_metadata_dir below. This folder should be
; a local folder that is not accessible from a browser.
; DEFAULT: "false"
;album_art_store_disk = "true"

; Local Metadata Directory
; This define a local metadata directory OUTSIDE the web path with write access.
; This is used to store heavy data if enabled (album art, waveforms, etc)
; DEFAULT: none
;local_metadata_dir = "/var/www/metadata"

Why should that need to be changed??  I think I am happy with the art in DB

**Reply by mitchray** [reply]
Doesn't need to, just if it was on disk then it would probably be a file permission issue

**Reply by rhauff1** [reply]
Running the catalog "Gather Art" does seem to work,  tried it on a couple new albums today.  I Assume it's using my metadata images??
